### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nishantwrp**
Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-
-

### Screenshots for the change

-
-

### CODE REVIEW

**Comment by Puranjay25** [comment]
This PR doesn't make any sense. It should be closed.

File: README.md [hunk 1]
```diff
@@ -53,29 +53,29 @@ Please get familiar with the different components of the project in order to be
...
-- If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
+- Before you join development, please set up the system on your local machine and go through the application completely. Press on any link/button you can find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
+- If you would like to work on an issue, drop in a comment at issue. If it is already assigned to someone, but there is no sign of any work being done, please free to fall in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
```
Comment (pranav1698) [inline README.md]: why are you changing drop to fall? drop makes more sense

File: README.md [hunk 2]
```diff
@@ -53,29 +53,29 @@ Please get familiar with the different components of the project in order to be
...
 
 **Join the development**
 
-- Before you join development, please set up the system on your local machine and go through the application completely. Press on any link/button you can find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or to you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
-- If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
+- Before you join development, please set up the system on your local machine and go through the application completely. Press on any link/button you can find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
```
Comment (pranav1698) [inline README.md]: Again, unjustified change

File: README.md [hunk 3]
```diff
@@ -53,29 +53,29 @@ Please get familiar with the different components of the project in order to be
...
+- When you make very very minor changes to a PR of yours (like for example fixing a failing Travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterward so that you don't have an absurd number of commits for a minimal fix. (Learn how to squash at https://davidwalsh.name/squash-commits-git )
+- When you're submitting a PR for a UI-related issue, it would be awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers, and you'll also get reviews quicker.
 
 **Feature Requests and Bug Reports**
 
-- When you file a feature request or when you are submitting a bug report to the [Issue tracker](https://github.com/fossasia/badgeyay/issues), make sure you add steps to reproduce it. Especially if that bug is some weird/rare one.
+- When you file a feature request or when you are submitting a bug report to the [Issue tracker](https://github.com/fossasia/badgeyay/issues), make sure you add steps to reproduce it especially if that bug is some weird/rare one.
```
Comment (pranav1698) [inline README.md]: Not required

**Pull request closed without merging**